### DISCUSSION TITLE
How I can inject i18n inside my Pinia model?

### DISCUSSION THREAD

**Discussion by zozulinskyi** [CodeDredd/pinia-orm#908] [Q&A] [answered]
Hi everyone.
Any idea how I can do this?

I have next object structure in my model:
```
{ id: 12, names: { en: 'France', uk: 'Франція' } },
{ id: 13, names: { en: 'Germany', uk: 'Німеччина' } },
```

and I need show country by user language.
I can do something like that, but I am not sure that is a good idea.
```
get name(): string {
    const { locale } = useI18n();
    return this.names[locale.value];
}
```

**Comment by CodeDredd** [comment, marked as answer]
I am trying to make best practices for such things in (#999). It's not wrong what you do.
